stories/topics/ref-aap-aws-uninstall.adoc

```diff
@@ -0,0 +1,35 @@
+[id="ref-aws-uninstall"]
+
+= Uninstall
+
+The following document guides how to uninstall {PlatformNameShort} from {Market}. If extension nodes were deployed, a prerequisite step must be taken to remove the extension nodes before uninstalling the stack.
+
+== Uninstall extension nodes
+
+If extension nodes were installed, follow these steps to remove them before continuing with the uninstall. If extension nodes were not installed, skip these steps.
+
```

Comment:
.Procedure

Reply:
Added! I renamed this doc from ref to proc as well to reflect that. I can undo this as well if this was unnecessary